docs/program/apis/data-client.md

```diff
@@ -74,6 +74,66 @@ The data client API supports the following methods (among [others](https://pytho
 
 {{< readfile "/static/include/services/apis/data-client.md" >}}
 
+### TabularDataByFilter
+
+Retrieve optionally filtered tabular data from the [Viam app](https://app.viam.com).
+Find your tabular data under the **Sensors** subtab of the app's [**Data** tab](https://app.viam.com/data).
```

Comment:
Not quite what I meant. If they are looking at the API, we probably shouldn't direct them to look in the app - kinda feels like they came here to use the API. So I think telling them they can find the tabular data there seems better.
```suggestion
You can also find your tabular data under the **Sensors** subtab of the app's [**Data** tab](https://app.viam.com/data).
```

Reply:
Ok sounds good to me! I figured ppl might want to first see what files they are going to download